tutorials/darcy_thermo_mech/doc/content/systems/mooseobject.md

```diff
@@ -60,58 +60,3 @@ CustomObject::doSomething()
   return _scale;
 }
 ```
-
```

Comment:
This deletion makes me happy

Reply:
Wait...why is this deleted? This looks correct and current... AD objects return ADReals while non-AD objects return Reals